lib/facility.rb

```diff
@@ -1,14 +1,52 @@
 class Facility
   attr_reader :name, :address, :phone, :services
 
-  def initialize(name, address, phone)
+  def initialize(name, address, phone, services)
     @name = name
     @address = address
     @phone = phone
-    @services = []
+    @services = services
   end
 
   def add_services(service)
     @services << service
   end
+
+  def register_vehicles(vehicle)
+    case vehicle.type
```

Comment:
I don't believe Vehicle has a method or attribute of .type. There was a plate_type potentially though.